echo.go

```diff
@@ -873,8 +876,11 @@ func (ln tcpKeepAliveListener) Accept() (c net.Conn, err error) {
 	return
 }
 
-func newListener(address string) (*tcpKeepAliveListener, error) {
-	l, err := net.Listen("tcp", address)
+func newListener(address, network string) (*tcpKeepAliveListener, error) {
+	if network != "tcp" && network != "tcp4" && network != "tcp6" {
```

Comment:
Maybe it would be better to leave this check to Go `net` pkg instead? Why limit options here?

Reply:
Sorry, I answered this some days ago, but I forgot to publish it 😞

Reply:
The other options are "unix" or "unixpacket", I guess those options doesn't fit here
https://golang.org/pkg/net/#Listen

Reply:
Reasonable. Thank you.